src/node/jwt_key_auto_refresh.h

```diff
@@ -274,6 +276,9 @@ namespace ccf
       jwt_issuers->foreach([this, &ca_cert_bundles](
                              const JwtIssuer& issuer,
                              const JwtIssuerMetadata& metadata) {
+        // Increment attempts
+        attempts++;
```

Comment:
Shouldn't this happen only when auto_refresh is enabled for the issuer? Otherwise there will definitely be a mismatch between attempts and successes which may be misleading if multiple issuers are used where only some of them have auto_refresh enabled.

Reply:
Good point. I'll submit a new PR.